board/OrangePi-Plus2E/overlay/boot/loader.conf

```diff
@@ -0,0 +1 @@
+fdt_file=orangepi-plus-2e.dtb
```

Comment:
VGA ? You mean HDMI right ?
I'm not sure that uboot support H3 HDMI controller yet.